pkg/tenant/projects.go

```diff
@@ -76,21 +118,30 @@ func (s *Server) TenantProjectCreate(c *gin.Context) {
 	c.JSON(http.StatusCreated, out)
 }
 
+// ProjectList retrieves all projects assigned to an organization
+// and returns a 200 OK response.
+//
+// Route: /projects
 func (s *Server) ProjectList(c *gin.Context) {
-	// The following TODO task items will need to be
-	// implemented for each endpoint.
+	// TODO: Fetch the project's tenant ID from key.
+	var tenantID ulid.ULID
+
+	// Get projects from the database and return a 500 response
+	// if not successful.
+	if _, err := db.ListProjects(c.Request.Context(), tenantID); err != nil {
+		log.Error().Err(err).Msg("could not fetch projects from the database")
+		c.JSON(http.StatusInternalServerError, api.ErrorResponse("could not fetch projects from the database"))
+		return
+	}
+
+	// Build the response.
+	out := &api.ProjectPage{Projects: make([]*api.Project, 0)}
 
-	// TODO: Add authentication and authorization middleware
-	// TODO: Identify top-level info
-	// TODO: Parse and validate user input
-	// TODO: Perform work on the request, e.g. database interactions,
-	// sending notifications, accessing other services, etc.
+	project := &api.Project{}
 
-	// Return response with the correct status code
+	out.Projects = append(out.Projects, project)
```

Comment:
Similar to `MemberList` we need to populate the list of returned `api.Project` here.

Reply:
Same as the comment above about `MemberList`.